FWCore/ParameterSet/python/Types.py

```diff
@@ -659,7 +659,15 @@ def _setValues(self,moduleLabel,productInstanceLabel='',processName=''):
         self.__moduleLabel = moduleLabel
         self.__productInstance = productInstanceLabel
         self.__processName=processName
-        if -1 != moduleLabel.find(":"):
+        if isinstance(moduleLabel, tuple):
+            self.__moduleLabel = moduleLabel[0] if len(moduleLabel) > 0 else ""
+            if len(moduleLabel) > 1:
+                self.__productInstance = moduleLabel[1]
```

Comment:
There is no check that the value returned is a string

Reply:
> There is no check that the value returned is a string

I did not add an explicit check (because we don't have them elsewhere either), but now the tuple/list elements must be accepted by `str.join()`. Would that be sufficient?